tests/primer/primer_tool.py

```diff
@@ -112,11 +130,79 @@ def _handle_run_command(self) -> None:
         ) as f:
             json.dump(packages, f)
 
+    def _handle_compare_command(self) -> None:
+        with open(self.config.base_file, encoding="utf-8") as f:
+            main_dict: PackageMessages = json.load(f)
+        with open(self.config.new_file, encoding="utf-8") as f:
+            new_dict: PackageMessages = json.load(f)
+
+        final_main_dict: PackageMessages = {}
+        for package, messages in main_dict.items():
+            final_main_dict[package] = []
+            for message in messages:
+                try:
+                    new_dict[package].remove(message)
+                except ValueError:
+                    final_main_dict[package].append(message)
+
+        self._create_comment(final_main_dict, new_dict)
+
+    def _create_comment(
+        self, all_missing_messages: PackageMessages, all_new_messages: PackageMessages
+    ) -> None:
+        comment = ""
+        for package, missing_messages in all_missing_messages.items():
+            new_messages = all_new_messages[package]
+            package_data = self.packages[package]
+
+            if not missing_messages and not new_messages:
+                continue
+
+            comment += f"\n\n**Effect on [{package}]({self.packages[package].url}):**\n"
+
+            if missing_messages:
+                comment += "The following messages are no longer emitted:\n"
+            count = 1
+            for message in missing_messages:
+                comment += (
+                    f"{count}) {message['symbol']} on line {message['line']} here:\n"
```

Comment:
For simple messages where the surrounding context is irrelevant, or messages that actually know the relevant context, like `redefined-outer-name`, everything relevant is already in the message text.  So what do you think about using the message text?
```suggestion
                    f"{count}) {message['symbol']}: {message['message']}:\n"
```

Reply:
Perhaps we should format it with italics, but I like the suggestion!